pkg/metrics/metrics.go

```diff
@@ -234,12 +236,13 @@ func EmitSubMetric(sub *olmv1alpha1.Subscription) {
 	if sub.Spec == nil {
 		return
 	}
-	SubscriptionSyncCount.WithLabelValues(sub.GetName(), sub.Status.InstalledCSV, sub.Spec.Channel, sub.Spec.Package).Inc()
+	SubscriptionSyncCount.WithLabelValues(sub.GetName(), sub.Status.InstalledCSV, sub.Spec.Channel, sub.Spec.Package, string(sub.Spec.InstallPlanApproval)).Inc()
```

Comment:
I'm not sure if `string(sub.Spec.InstallPlanApproval)` is the right way to go about this. 
Ideally, there should have been a String() defined where `InstallPlanApproval` was defined. But since making this change now would require API version bump, maybe we can add it in this file: 

```golang
func InstallPlanApproval(ip v1alpha1.Approval) string {
     return string(ip)
}
```
and then use that method everywhere instead.

Reply:
That would return an error:
`undefined: "github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/listers/operators/v1alpha1".Approval`

Approval is defined [here](https://github.com/operator-framework/api/blob/master/pkg/operators/installplan_types.go)